src/sentry/migrations/0300_deploy_date_finished_index.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.28 on 2022-07-11 03:24
+
+import django.utils.timezone
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
```

Comment:
This will have to be `is_dangerous =. True` so that ops can run it separately. Since this table has 3B rows, adding this index will take too long to run.

Reply:
Just a correction here - table is 24mm rows, looked at the wrong tab. Still best to create this as is_dangerous though